dataplane/src/pipeline/test_utils.rs

```diff
@@ -99,3 +101,13 @@ pub fn build_test_ipv4_packet(ttl: u8) -> Result<Packet<TestBuffer>, InvalidPack
 
     Packet::new(buffer)
 }
+
+/// Initializes tracing for tests that emit logs
+pub fn init_test_tracing() {
```

Comment:
I see what you are doing here (I did the same thing a little while back).

Then I found  [tracing_test](https://docs.rs/tracing-test/latest/tracing_test/) (which is already a workspace dependency).  You can skip this using `#[traced_test]` as an annotation on the test functions.

The [trap with this approach is described in the docs](https://docs.rs/tracing-test/latest/tracing_test/#rationale--why-you-need-this)

Reply:
Ack! Thanks @daniel-noland , will remove the method.